extensions/amp-story-audio-sticker/0.1/amp-story-audio-sticker.js

```diff
@@ -1,21 +1,97 @@
 import * as Preact from '#core/dom/jsx';
 import {Layout_Enum} from '#core/dom/layout';
+import {
+  closestAncestorElementBySelector,
+  scopedQuerySelector,
+} from '#core/dom/query';
 
 import {Services} from '#service';
 
 import {CSS} from '../../../build/amp-story-audio-sticker-0.1.css';
-import {Action} from '../../amp-story/1.0/amp-story-store-service';
+import {
+  Action,
+  StateProperty,
+} from '../../amp-story/1.0/amp-story-store-service';
 
 const TAG = 'amp-story-audio-sticker';
 
+/**
+ * Sticker pretap tag
+ * @const {string}
+ */
+const STICKER_PRETAP_TAG = 'amp-story-audio-sticker-pretap';
+/**
+ * Sticker posttap tag
+ * @const {string}
+ */
+const STICKER_POSTTAP_TAG = 'amp-story-audio-sticker-posttap';
+
+/**
+ * The gstatic URL prefix of the sticker assets.
+ * @const {string}
+ */
+const ASSET_URL_PREFIX = 'https://www.gstatic.com/amphtml/stamp/audio-sticker/';
+
+/** @const {!Object}
+ * List of default stickers.
+ */
+const DEFAULT_STICKERS = {
+  'audio-cloud': {
+    width: '360',
+    height: '233',
+    pretapUrl: ASSET_URL_PREFIX + 'audio-cloud-pretap.png',
+    posttapUrl: ASSET_URL_PREFIX + 'audio-cloud-posttap.png',
+  },
+  'cat-sticker': {
+    width: '282',
+    height: '226',
+    pretapUrl: ASSET_URL_PREFIX + 'cat-sticker-pretap.png',
+    posttapUrl: ASSET_URL_PREFIX + 'cat-sticker-posttap.gif',
+  },
+  'loud-speaker': {
+    width: '380',
+    height: '380',
+    pretapUrl: ASSET_URL_PREFIX + 'loud-speaker-pretap.png',
+    posttapUrl: ASSET_URL_PREFIX + 'loud-speaker-posttap.png',
+  },
+  'tape-player': {
+    width: '260',
+    height: '220',
+    pretapUrl: ASSET_URL_PREFIX + 'tape-player-pretap.png',
+    posttapUrl: ASSET_URL_PREFIX + 'tape-player-posttap.gif',
+  },
+};
+
+/** Fallback default sticker. It would be used if both of below happen:
+ * 1) no default sticker is specified
+ * 2) the custom sticker data is not provided completely.
+ * @const {string}
+ */
+const FALLBACK_DEFAULT_STICKER = 'cat-sticker';
+
+/**
+ * The number of milliseconds to wait before hiding the sticker after the story is unmuted.
+ * @const {number}
+ */
+export const HIDE_STICKER_DELAY_DURATION = 4000;
```

Comment:
This doesn't need to exist here. You can hardcode the const for the timer values in the tests so that the module doesn't need to export anything.